restservice/src/main/java/org/itevents/mapper/EventMapper.java

```diff
@@ -15,43 +15,33 @@
             @Result(property = "regLink", column = "reg_link"),
             @Result(property = "address", column = "address"),
             @Result(property = "contact", column = "contact"),
-            @Result(property = "location", column = "id", javaType = Location.class, one=@One(select="org.itevents.mapper.LocationMapper.selectLocation"))
+            @Result(property = "location", column = "id", javaType = Location.class,
+                    one=@One(select="org.itevents.mapper.LocationMapper.selectLocation"))
     })
     @Select("SELECT id, title, event_date, create_date, reg_link, address, contact FROM events WHERE id = #{id}")
     Event getEvent(int id);
 
-    @Results(value = {
-            @Result(property = "id", column = "id"),
-            @Result(property = "title", column = "title"),
-            @Result(property = "eventDate", column = "event_date"),
-            @Result(property = "createDate", column = "create_date"),
-            @Result(property = "regLink", column = "reg_link"),
-            @Result(property = "address", column = "address"),
-            @Result(property = "contact", column = "contact"),
-            @Result(property = "location", column = "id", javaType = Location.class, one=@One(select="org.itevents.mapper.LocationMapper.selectLocation"))
-    })
+    @ResultMap(value = "getEvent-int")
     @Select("SELECT id, title, event_date, create_date, reg_link, address, contact FROM events")
     List<Event> getAllEvents();
 
-    @Results(value = {
-            @Result(property = "id", column = "id"),
-            @Result(property = "title", column = "title"),
-            @Result(property = "eventDate", column = "event_date"),
-            @Result(property = "createDate", column = "create_date"),
-            @Result(property = "regLink", column = "reg_link"),
-            @Result(property = "address", column = "address"),
-            @Result(property = "contact", column = "contact"),
-            @Result(property = "location", column = "id", javaType = Location.class, one=@One(select="org.itevents.mapper.LocationMapper.selectLocation"))
-    })
-    @Select("SELECT id, title, event_date, create_date, reg_link, address, contact FROM events WHERE ST_DWithin(point::geography, ST_MakePoint(#{location.longitude},#{location.latitude})::geography, #{radius})")
-    List<Event> getEventsInRadius(@Param("location") Location location, @Param("radius") int radius);
+    @ResultMap(value = "getEvent-int")
+    @Select("SELECT id, title, event_date, create_date, reg_link, address, contact FROM events" +
+            " WHERE ST_DWithin(point)::geography," +
```

Comment:
There is redundant brace in SELECT. Should be something like this: ... WHERE ST_SWithin(point::geography,...
Please careful check similars